src/ui/hexedit.cc

```diff
@@ -211,6 +218,25 @@ HexEdit::HexEdit(FileBlobModel *dataModel, QItemSelectionModel *selectionModel,
     copyMenu->addAction(copyAction);
   }
 
+  auto pasteMenu = menu_.addMenu("Paste as");
```

Comment:
Should be `Paste from`, not `Paste as` ("pasting as hex" doesn't hex-encode clipboard contents and then paste).